dashboard/db/migrate/20210327012046_create_lessons_opportunity_standards.rb

```diff
@@ -0,0 +1,14 @@
+class CreateLessonsOpportunityStandards < ActiveRecord::Migration[5.2]
+  def change
+    create_join_table :lessons, :standards, table_name: :lessons_opportunity_standards do |t|
+      t.index [:lesson_id, :standard_id], unique: true
+      t.index [:standard_id, :lesson_id]
+    end
+    add_column :lessons_opportunity_standards, :id, :primary_key
```

Comment:
I had trouble with a join table with an id but I think you address that in the next PR with `has_many` and `dependent: :destroy`

Reply:
yeah it probably would have been clearer to create this without the `create_join_table` syntax and just created this as a normal table instead. I'm not aware of any difference in the outcome aside from what appears inside the migration file, though.